modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses label-based rules to determine if a pod should be injected with a sidecar. If a namespace has both the `istio-injection` label and the `istio.io/rev` label, the `istio-injection` label takes precedence, treating the namespace as part of the default revision.
+
+The sidecar injector follows the following logic:
+
+* If either the `istio-injection` label or the `sidecar.istio.io/inject` label is set to disabled, the pod is not injected.
+* If any of the following labels are enabled the pod is injected with a sidecar: `istio-injection`, `sidecar.istio.io/inject`, or `istio.io/rev`.
+
+.Prerequisites
+
+* The {SMProduct} operator is installed
+* An {istio} Container Network Interface (CNI) resource is created
+
+.Procedure
+
+. Create namespace called `istio-system`: 
++
+[source,terminal]
+----
+$ oc create ns istio-system
+----
+
+. Create a YAML file named `istio.yaml` with the name as `default`:
++
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  version: v1.23.0
+----
+
+. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+. Wait for {istio} to get ready by running the following command:
++
+[source,terminal]
+----
+$ oc wait --for=condition=Ready istios/default -n istio-system
+----
+
+. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/sleep/sleep.yaml
+----
+
+. Verify that the deployment has a single container by running the folowing command:
++
+[source,terminal]
+----
+$ oc get deployment -o wide
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME    READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES            SELECTOR
+sleep   1/1     1            1           16s   sleep        curlimages/curl   app=sleep
+----
+
+. Verify that the pod has a single container by running the folowing command:
++
+[source,terminal]
+----
+$ oc get pod -l app=sleep
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                     READY   STATUS    RESTARTS   AGE
+sleep-5577c64d7c-ntn9d   1/1     Running   0          16s
+----
+
+. Add the label `istio-injection=enabled` to the `default` namespace:
```

Comment:
I'd rephrase the last part to emphasize that the label should be added to the namespace in which the sleep deployment was deployed (which in our case is `default`). 

We need to make sure that the reader doesn't think that they always need to add the label to the `default` namespace.

Reply:
Changed it to: . Add the label `istio-injection=enabled` to the namespace: